app/controllers/catalog_controller_decorator.rb

```diff
@@ -39,7 +39,7 @@
   # see: https://github.com/scientist-softserv/adventist-dl/blob/97bd05946345926b2b6c706bd90e183a9d78e8ef/app/controllers/catalog_controller.rb#L38-L40
   config.index_fields.keys.each do |key|
     next if key == 'all_text_timv'
-    next if key == 'file_format_tsimv'
+    next if key == 'file_set_text_tsimv'
```

Comment:
Why are we removing `file_format_tsimv`?  Would it be a problem to keep it?

Reply:
it was a typo actually